spec/fixtures/fales/bloch.xml

```diff
@@ -79,6 +79,7 @@
             <genreform source="aat">Postcards.</genreform>
             <genreform source="aat">Posters.</genreform>
             <subject source="lcsh">Zines.</subject>
+            <subject source="lcsh">World War ||</subject>
```

Comment:
No,  I didn't realized it was capital i's. Will change

On Tue, Jan 20, 2015 at 4:34 PM, barnabyalter <notifications@github.com>
wrote:

> In spec/fixtures/fales/bloch.xml
> <https://github.com/NYULibraries/findingaids/pull/27#discussion-diff-23258055>
> :
>
> > @@ -79,6 +79,7 @@
> >              <genreform source="aat">Postcards.</genreform>
> >              <genreform source="aat">Posters.</genreform>
> >              <subject source="lcsh">Zines.</subject>
> > +            <subject source="lcsh">World War ||</subject>
>
> @ekate <https://github.com/ekate> The acceptance criteria says "World War
> II" with capital i's, but it looks like these are bar characters instead.
> Is there a reason you used these characters?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NYULibraries/findingaids/pull/27/files#r23258055>.
>



-- 
Ekaterina Pechekhonova
Digital Library Programmer/Analyst
New York University
Libraries
email: ekate@nyu.edu
phone: 212-992-9993